components/advanced_create_post/advanced_create_post.tsx

```diff
@@ -1255,6 +1273,40 @@ class AdvancedCreatePost extends React.PureComponent<Props, State> {
         }]);
     }
 
+    handleRemovePriority = () => {
+        this.handlePostPriorityApply({priority: undefined});
+    }
+
+    handlePostPriorityApply = ({priority}: {priority: PostPriority|undefined}) => {
```

Comment:
why is the parameter wrapped in an object?

Wouldn't it be easier to just use it like this:

```suggestion
    handlePostPriorityApply = (priority?: PostPriority) => {
```

Reply:
@michelengelen in the near future we are going to add extra options, like message ack, and persistent notifications, and passing an object would help with all.

Reply:
still typing it with `?` you can drop the `|undefined` union